app/models/post.rb

```diff
@@ -0,0 +1,13 @@
+class Post < ApplicationRecord
+  belongs_to :author, class_name: 'User'
+  has_many :comments
+  has_many :likes
+
+  def update_posts_counter
+    author.update(posts_counter: author.posts_counter + 1)
+  end
```

Comment:
- [ ] [OPTIONAL] you can use the pre-made methods in rails `author.increment!(:posts_counter)` to increment post counter